package.json

```diff
@@ -15,6 +15,7 @@
     "@wordpress/babel-preset-default": "^4.0.0",
     "@wordpress/eslint-plugin": "^2.0.0",
     "@wordpress/jest-preset-default": "^4.0.0",
+    "@wordpress/server-side-render": "file:gutenberg/packages/server-side-render",
```

Comment:
we usually don't need to add `@wordpress` package dependencies here unless they're used for the build/linter.